qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/OpenConceptLabSteps.java

```diff
@@ -0,0 +1,134 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.automation;
+ 
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import java.util.List;
+ 
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.page.ConfigureMetadataPage;
+import org.openmrs.contrib.qaframework.page.OpenConceptLabPage;
+import org.openmrs.contrib.qaframework.page.OpenConceptLabSuccessPage;
+import org.openmrs.contrib.qaframework.page.SubscriptionPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+ 
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+ 
+public class OpenConceptLabSteps extends Steps  {
+    private OpenConceptLabPage openconceptlabpage;
+    private ConfigureMetadataPage configureMetadataPage;
+    private static final String releasedDictionaryUrl = "https://api.staging.openconceptlab.org/users/openmrs/collections/TD-ecly/65725685/";
+    private static final String tokenUrl = "bd022mad6d3df24f5c42ewewa94b53a23edf6eee7r";
+    private static final String newDictionaryUrl = "http://api.openconceptlab.com/orgs/CIEL/sources/CIEL/";
+    private static final String newTokenUrl = "bd022mad6d3df24f5c42ewewa94b53a23edf6fff7r";
+    private OpenConceptLabSuccessPage openConceptLabSuccessPage;
```

Comment:
What is the use of the newDictionaryUrl and newTokenUrl?

Reply:
@Irenyak1 thanks for the review. My intention for the newDictionaryUrl and newTokenUrl was to be used when the user edits either of the 2.

Reply:
> @Irenyak1 thanks for the review. My intention for the newDictionaryUrl and newTokenUrl was to be used when the user edits either of the 2
@jwnasambu  ok then, you may need to use them as opposed you the urls you put directly.